cli/internal/run/global_hash.go

```diff
@@ -88,7 +88,7 @@ func calculateGlobalHash(rootpath fs.AbsolutePath, rootPackageJSON *fs.PackageJS
 
 	globalFileHashMap, err := hashing.GetHashableDeps(rootpath, globalDepsPaths)
 	if err != nil {
-		return "", fmt.Errorf("error hashing files. make sure that git has been initialized %w", err)
+		return "", fmt.Errorf("error hashing files. make sure that git has been initialized: %w", err)
```

Comment:
I think this error message is no longer correct. We aren't requiring git anymore.

Reply:
Good catch